config/locales/oc.yml

```diff
@@ -206,8 +206,62 @@ oc:
   authorize_follow:
     error: O planhèm, i a agut una error al moment de cercar lo compte
     follow: Sègre
-    prompt_html: 'Avètz (<strong>%{self}</strong>) demandat de sègre :'
```

Comment:
Well, normally the subject of the verb is included in the conjugation, I wondered if I could put the emphasis on the "avètz" instead.